apps/portals/e2e/explore.spec.ts

```diff
@@ -0,0 +1,362 @@
+import { Locator, Page, expect, test } from '@playwright/test'
+import { defaultExpectTimeout } from '../playwright.config'
+import exploreConfig from './configs/exploreConfig'
+import { dismissBanners } from './helpers/banners'
+import { getPortal } from './helpers/portalInfo'
+
+const portalExploreConfig = exploreConfig[getPortal()]
+
+const goToExploreTab = async (page: Page, exploreTab: string) => {
+  await test.step(`go to Explore/${exploreTab}`, async () => {
+    await test.step('navigate to explore page', async () => {
+      const cleanUrl = exploreTab.replace(' ', '%20')
+      await page.goto(`/Explore/${cleanUrl}`)
+    })
+
+    await test.step('explore page has loaded', async () => {
+      await expect(
+        page.getByRole('main').getByText('Explore', { exact: true }),
+      ).toBeVisible()
+      await expect(page.getByLabel('Explore Sections')).toBeVisible()
+      await expect(page.getByRole('tab', { name: exploreTab })).toBeVisible()
+    })
+
+    await dismissBanners(page)
+  })
+}
+
+const expectCards = async (page: Page) => {
+  await test.step('expect cards', async () => {
+    const { cards, nCards } = await test.step('expect cards list', async () => {
+      const cards = page.locator('.SRC-portalCard')
```

Comment:
When text-based selectors would be specific to each Portal, use class names to generalize locators across Portals